sources/main.cpp

```diff
@@ -19,6 +20,8 @@
 #include <DYNInitXml.h>
 #include <DYNIoDico.h>
 #include <boost/filesystem.hpp>
+#include <boost/property_tree/json_parser.hpp>
```

Comment:
Why do you need the JSON parser here ?

Reply:
Fixed. Neither `json_parser` nor `ptree` were used.